src/syrupy/utils.py

```diff
@@ -14,3 +18,33 @@ def walk_snapshot_dir(root: str) -> Generator[str, None, None]:
             continue
         if not filepath.name.startswith(".") and filepath.is_file():
             yield str(filepath)
+
+
+class FailedToLoadModuleMember(Exception):
+    pass
+
+
+def load_module_member_from_path(path: str) -> Any:
+    path_parts = path.split(".")
+
+    module_name = ".".join(path_parts[:-1])
+    module_member_name = path_parts[-1:][0]
+
+    if not module_name:
+        raise FailedToLoadModuleMember(
+            f"Cannot load member '{module_member_name}' without module path"
+        )
+
+    try:
+        module = import_module(module_name)
+    except ModuleNotFoundError:
+        raise FailedToLoadModuleMember(f"Module '{module_name}' does not exist.")
+
+    try:
+        module_member = getattr(module, module_member_name)
```

Comment:
Can't `import_module` directly import a single export in a module. Or is this to provide better error messages?

Reply:
Just tried, guess it can't and I can't see any clean way to do it